### PR TITLE
chore(docs): Fix Environment Variables and Best Practices 

### PR DESCRIPTION
**Pull request by laurapacilio**
- Put environment variables in a table and revise language

### CODE REVIEW

**Comment by laurapacilio** [comment]
@DanielMSchmidt and @ansgarm I forget exactly what we need to do to get this live on the site....do I need to merge to stable-website, too?

File: website/docs/cdktf/create-and-deploy/environment-variables.mdx [hunk 1]
```diff
@@ -7,48 +7,17 @@ description: >-
...
 
-Use these environment variables to configure CDK for Terraform behavior.
+Use these environment variables to configure CDK for Terraform (CDKTF) behavior.
+
+| Variable Name | Description | Default |
+| ------------- | ----------- | -------|
+| CDKTF_DISABLE_PLUGIN_CACHE_ENV | If set to `true` or `1`, CDKTF disables the [plugin cache](/cli/config/config-file#provider-plugin-cache). Otherwise, CDKTF creates the plugin cache directory in `~/.terraform.d/plugin-cache` if it is not already present. |  |
+| CDKTF_HOME | Where CDKTF should store cache and configuration files. | `~/.cdktf` |
+| CDKTF_LOG_FILE_DIRECTORY | The directory path where CDKTF should create `cdktf.log` and print logs at the `debug` level. If not set, CDKTF writes logs to standard out at the level specified in `CDKTF_LOG_LEVEL`. | standard out |
+| CDKTF_LOG_LEVEL | Controls how much log information CDKTF prints to the console. You can set it to one of the following values: `all`, `debug`, `info`, `warn`, `error`, `fatal`, `off`. | `debug` |
```
Comment (laurapacilio) [inline website/docs/cdktf/create-and-deploy/environment-variables.mdx:17]: I made an assumption that the default is `debug` PLEASE correct me if I'm wrong!

File: website/docs/cdktf/create-and-deploy/environment-variables.mdx [hunk 2]
```diff
@@ -7,48 +7,17 @@ description: >-
...
+| CHECKPOINT_DISABLE | If set to any value, CDKTF will not send [telemetry](/cdktf/telemetry) data. |   |
+| DISABLE_VERSION_CHECK | If set to `true` or `1`, CDKTF will not perform a version check. If it is not set, CDKTF verifies that your installed library version matches the CLI version. If they do not match, CDKTF displays a warning. |  |
+| HTTP_PROXY or http_proxy| The proxy that CDKTF should use for all HTTP requests. |  |
+| TERRAFORM_BINARY_NAME | The Terraform binary that CDKTF should use. | The Terraform binary in path |
```
Comment (laurapacilio) [inline website/docs/cdktf/create-and-deploy/environment-variables.mdx:21]: I don't really understand what we mean by "terraform binary in path - can you explain?

File: website/docs/cdktf/create-and-deploy/environment-variables.mdx (hunk 1 above)
Comment (DanielMSchmidt) [reply website/docs/cdktf/create-and-deploy/environment-variables.mdx:17]: The default is `warn`

File: website/docs/cdktf/create-and-deploy/environment-variables.mdx (hunk 2 above)
Comment (DanielMSchmidt) [reply website/docs/cdktf/create-and-deploy/environment-variables.mdx:21]: Basically the path is a list of directories user set in their shell config where the shell looks for executable programs to run. Without a path users would need to write e.g. `/usr/local/bin/terraform plan` but if you append `/usr/local/bin/` to your path you can just write `terraform plan` in the shell instead. If you need to use different versions of a program at the same point in time (e.g. you have an old 0.12 terraform project and a new 1.0 terraform project) users normally rename the old binary, so they write `terraform0.12 plan` to access the old binary.

File: website/docs/cdktf/create-and-deploy/environment-variables.mdx (hunk 1 above)
Comment (laurapacilio) [inline website/docs/cdktf/create-and-deploy/environment-variables.mdx:17]:
```suggestion
| CDKTF_LOG_LEVEL | Controls how much log information CDKTF prints to the console. You can set it to one of the following values: `all`, `debug`, `info`, `warn`, `error`, `fatal`, `off`. | `warn` |
```

Comment (laurapacilio) [reply website/docs/cdktf/create-and-deploy/environment-variables.mdx]: fixed!

File: website/docs/cdktf/create-and-deploy/environment-variables.mdx (hunk 2 above)
Comment (laurapacilio) [reply website/docs/cdktf/create-and-deploy/environment-variables.mdx:21]: OHHHHHH okay - you think developers who aren't me will just know this, right? If so, I'll resolve this comment. Thank you for explaining :)

Comment (DanielMSchmidt) [reply website/docs/cdktf/create-and-deploy/environment-variables.mdx:21]: Yeah they should know this

**Review by DanielMSchmidt — APPROVED** [review]

**Comment by laurapacilio** [comment]
@DanielMSchmidt and @ansgarm if you feel comfortable with this, would you mind merging it? I don't have permissions :).

**Comment by DanielMSchmidt** [comment]
Updated the stable website branch with this :)

**Pull request merged**